src/Text/Pandoc/Writers/LaTeX.hs

```diff
@@ -1497,7 +1497,8 @@ citationsToBiblatex (c:cs)
   where grouper prev cit = case prev of  
          ((CiteGroup oPfx oSfx ids):rest)
              | null oSfx && null pfx -> (CiteGroup oPfx sfx (cid:ids)):rest
-             | null pfx -> (CiteGroup oPfx sfx [cid]):prev 
+             | null pfx && null sfx -> (CiteGroup oPfx [] [cid]):prev 
+             | null oPfx && null sfx && null pfx -> (CiteGroup [] [] [cid]):prev 
          _ -> (CiteGroup pfx sfx [cid]):prev
          where pfx = citationPrefix cit 
```

Comment:
Line 1501 is superfluous since the same case will be handled the same way by line 1502, correct?
 I'm puzzled about line 1500.  Isn't this again duplicating a prefix, which is what we wanted to avoid?  Do you have a test case that exercises this condition, and what does it look like?

Reply:
No it isn't, it passes the test cases because of the ``optargs`` code in ``citeArgumentsList``, however you're right line 1501 is not needed

Reply:
checked over it again, it's not because of that it's because of the prefix before was null in the test, looking at it the tests only passes because of that,  this wouldn't work for some thing like:
``[e.g. @a1, ch.3 and elsewhere;@a2; but also @a4;@a5]`` I'll give this another try.